### PR TITLE
DM-7050: Migration to Google Cloud SQL

### PR DESCRIPTION
**Pull request by jonathansick**
This PR adds docs to facilitate a migration from sqlite in production to Google Cloud SQL.

- Changes DB schema because of a length limit for unicode columns that have a `unique` constraint on MySQL.
- Enables alembic features for doing non-trivial migrations of sqlite DBs.
- Adds a command to create a DB with tables (`run.py createdb`)
- Adds a command to copy a DB (`run.py copydb`), based on https://github.com/ei-grad/sqlacrossover
- Documentation features Google Cloud SQL in production

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Flask==0.10.1
 uWSGI==2.0.12
 Flask-SQLAlchemy==2.1
+PyMySQL==0.7.6
```
Comment (jmatt) [inline requirements.txt]: Happy to see the python only driver being used! Let's hope performance never forces the project to MySQLdb.

**Comment by jmatt** [comment]
Maybe consider removing the conversion code and leave it in the commit history since the DB is converted and it will never be used again.

**Comment by jmatt** [comment]
Looks good.

**Comment by jonathansick** [comment]
> Maybe consider removing the conversion code and leave it in the commit history since the DB is converted and it will never be used again.

The copydb command? I dunno, might be handy in case we move to RDS or something hosted on Nebula.

**Pull request merged**